locales/en/default.json

```diff
@@ -37,7 +37,9 @@
     },
     "dob": {
       "required": "Please enter a date of birth",
-      "british-date": "Please enter a date of birth in the correct format"
+      "british-date": "Please enter a date of birth in the correct format",
+      "past": "Please enter a date of birth that is not in the future",
```

Comment:
Ok cool - perfectly happy to wait for a better alternative...
Ideally I'd like a solution that gets the [valid from] date from the config so it doesn't have to be hard coded, but alas we can't really do this in plain old JSON.